spec/rails_helper.rb

```diff
@@ -34,6 +34,8 @@
 RSpec.configure do |config|
   # config taken directly from RSpec example in the DatabaseCleaner README
   config.before(:suite) do
+    `npm install` or raise 'could not install deps'
```

Comment:
@hoffmanc, this should be run after the deployment, and not as part of the suite.